thirdeye/thirdeye-frontend/app/pods/rootcause/controller.js

```diff
@@ -293,15 +297,20 @@ export default Ember.Controller.extend({
       this.setProperties({ hoverUrns: new Set(urns), hoverTimestamp: timestamp });
     },
 
+    onSessionChange(name, text) {
+      console.log('onSessionChange(name, text)');
+      this.setProperties({ sessionName: name, sessionText: text, sessionModified: true });
+    },
+
     onSessionSave() {
       const jsonString = JSON.stringify(this._makeSession());
 
       return fetch(`/session/`, { method: 'POST', body: jsonString })
         .then(checkStatus)
-        .then(res => this.set('sessionId', res));
+        .then(res => this.setProperties({ sessionId: res, sessionModified: false }));
```

Comment:
Since you're setting multiple variables, let's try to have each set in separate lines for readability. For example:
```
this.setProperties({
  sessionId: res,
  sessionModified: false
})
```

Reply:
fixed